app/models/user.rb

```diff
@@ -1,8 +1,12 @@
 class User < ActiveRecord::Base
   authenticates_with_sorcery!
 
-  validates :email, :password, presence: true
+  validates :email, presence: true
   validates :email, uniqueness: true, if: -> { email.present? }
+  validates :password, presence: true, on: :update, confirmation: true
```

Comment:
Yes I agree but at the moment we don't have a password forget functionality. I will create a separate issue for this after that is done we can drop the password confirmation, agreed?